### PR TITLE
Dpo refactoring

### PR DESCRIPTION
**Pull request by ssh16**
# More refactoring of the application

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/application/templates/document-provider-onboarding/src/forms/DocumentProviderApplication.ts [hunk 1]
```diff
@@ -65,6 +68,8 @@ export const DocumentProviderOnboarding: Form = buildForm({
               id: 'applicant.phoneNumber',
               name: m.applicantPhoneNumber,
               variant: 'tel',
+              format: '###-####',
+              placeholder: '000-0000',
```
Comment (hinriksteinar) [inline libs/application/templates/document-provider-onboarding/src/forms/DocumentProviderApplication.ts:72]: This makes it so we can only use icelandic phonenumbers. Might be some case where there is a phonenumber from different country ? Open for disscussion

File: libs/application/templates/document-provider-onboarding/src/forms/DocumentProviderApplication.ts [hunk 2]
```diff
@@ -156,11 +167,101 @@ export const DocumentProviderOnboarding: Form = buildForm({
...
+              id: 'applicant.address',
+              name: m.applicantAddress,
+              width: 'half',
+            }),
+            buildTextField({
+              id: 'applicant.zipCode',
+              name: m.applicantZipCode,
+              width: 'half',
+            }),
+            buildTextField({
```
Comment (hinriksteinar) [inline libs/application/templates/document-provider-onboarding/src/forms/DocumentProviderApplication.ts:196]: Was there a decision regarding which inputs should be half width ? I could imagine the email input can be to long for half width.

**Review by hinriksteinar — COMMENTED** [review]
Minor comments, but looks good !

File: libs/application/templates/document-provider-onboarding/src/forms/DocumentProviderApplication.ts (hunk 2 above)
Comment (ssh16) [reply libs/application/templates/document-provider-onboarding/src/forms/DocumentProviderApplication.ts:192]: We decided to make all the inputs full width :)

File: libs/application/templates/document-provider-onboarding/src/forms/DocumentProviderApplication.ts (hunk 1 above)
Comment (ssh16) [reply libs/application/templates/document-provider-onboarding/src/forms/DocumentProviderApplication.ts:72]: True, good point. We will have to think about that later on and maybe change it

**Review by hinriksteinar — APPROVED** [review]
LGTM

**Pull request merged**